flow-server/src/main/java/com/vaadin/flow/server/frontend/ThemeValidationUtil.java

```diff
@@ -128,16 +129,15 @@ private static void collectThemeJsonContentsInFrontend(Options options,
             String parentThemeName = parentThemeInFrontend.get();
             Optional<JsonObject> parentThemeJson = ThemeUtils.getThemeJson(
                     options.getFrontendDirectory(), parentThemeName);
-            if (parentThemeJson.isPresent()) {
-                collectThemeJsonContentsInFrontend(options, themeJsonContents,
-                        parentThemeName, parentThemeJson.get());
-            }
+            parentThemeJson.ifPresent(
+                    jsonObject -> collectThemeJsonContentsInFrontend(options,
+                            themeJsonContents, parentThemeName, jsonObject));
         }
 
         themeJsonContents.put(themeName, themeJson);
     }
 
-    private static boolean objectIncludesEntry(JsonValue jsonFromBundle,
+    protected static boolean objectIncludesEntry(JsonValue jsonFromBundle,
```

Comment:
It can probably be package protected, if it should be visible only for tests
```suggestion
    static boolean objectIncludesEntry(JsonValue jsonFromBundle,
```

Reply:
Okey, correct, even better. Thanks!